traffic_ops/traffic_ops_golang/types/types.go

```diff
@@ -158,6 +174,34 @@ func (tp *TOType) loadUseInTable() (error, error, string) {
 	return nil, nil, useInTable
 }
 
+func selectMaxLastUpdatedQuery(where, orderBy, pagination, where2, orderBy2, pagination2 string) string {
```

Comment:
The reason I did that is because of the way the "ParamColumns" method works. Its creating an alias for the main table and then passing that along to the dbHelpers function to create the "where" clause. If we have a deleted table, we would need to query against that table (with its own alias as well). For ex, this:

`SELECT max(t) from (
		SELECT max(last_updated) as t from type typ WHERE typ.id=12345
		UNION ALL
	SELECT max(last_updated) as t from deleted_type dtyp WHERE dtyp.id=12345
		) as res`
This is why I was thinking of splitting out the two. The only way I could think of was by changing the "ParamColumns" method, but that seemed like a more involved change. Is there a better way of doing this?

Reply:
The `UNION ALL` queries can have the same alias, they're effectively separate queries. So
```
`SELECT max(t) from (
		SELECT max(last_updated) as t from type typ ` + where + orderBy + pagination +
		` UNION ALL
	select max(last_updated) as t from deleted_type typ ` + where + orderBy + pagination +
		` ) as res`
```
Works fine

Reply:
👍